src/__tests__/unit/util/args.test.ts

```diff
@@ -35,28 +51,47 @@ import path = require('path');
 import {parseArgs} from '../../../util/args';
 import {ERRORS} from '../../../util/errors';
 
-import {STRINGS} from '../../../config';
+import {STRINGS, CONFIG} from '../../../config';
+const {impact} = CONFIG;
+const {HELP} = impact;
 
 const {CliInputError} = ERRORS;
 
 const {MANIFEST_IS_MISSING, FILE_IS_NOT_YAML} = STRINGS;
 
+const info = jest.spyOn(console, 'info').mockImplementation(() => {});
+
 describe('util/args: ', () => {
   const originalEnv = process.env;
 
   describe('parseArgs(): ', () => {
     it('throws error if there is no argument passed.', () => {
-      expect.assertions(2);
+      expect.assertions(5);
 
       process.env.result = 'error'; // used for mocking
 
       try {
```

Comment:
shouldn't try-catch blocks be in separate unit tests?

Reply:
I've combined all of them because the meaning is the same